### DISCUSSION TITLE
flowbite Modal

### DISCUSSION THREAD

**Discussion by thiagors-dev** [themesberg/flowbite#324] [Q&A] [answered]
Hello, I'm using flowbite with laravel and I followed the official website step-by-step : https://flowbite.com/docs/getting-started/laravel/

However, I can't get Modal to work. It returns this error in the console:

`Uncaught TypeError: Cannot read properties of null (reading 'getAttribute')
    at modal.js:122:35
    at NodeList.forEach (<anonymous>)
    at HTMLDocument.initModal (modal.js:119:54)`

Has anyone gone through this or something similar?

**Comment by mrtorks** [comment, marked as answer]
I think your data-toggle-modal might be wrong. do not add `#` to that variable. What I mean is lets say your modal has `id="testModal"`;

Do not set `data-toggle-modal="#testModal"` set it to this instead `data-toggle-modal="testModal"`
